packages/cli/src/services/public-api-key.service.ts

```diff
@@ -105,6 +109,12 @@ export class PublicApiKeyService {
 
 			if (!user) return false;
 
+			try {
+				this.jwtService.verify(providedApiKey);
+			} catch (e) {
+				return false;
+			}
```

Comment:
This is to guard against expired tokens, right? Let's check that the error is `TokenExpiredError` and return false in that case and rethrow in other cases. The token should always be valid as we have created it and store it in the DB. Otherwise we'd like to know if that's not the case, as there's a bug somewhere

Reply:
Rethrow the same excpetion. Something like

```ts
catch (e) {
  if (e instanceof TokenExpiredError) {
    return false
  }

  throw e;
}
```

Reply:
done!